.circleci/config.yml

```diff
@@ -116,12 +116,16 @@ jobs:
             circleci step halt
           fi
     - checkout
+    - restore_cache:
+        <<: *restore_cache_beam
     - restore_cache:
         keys:
           # Use same cache as the ingestion-beam job.
           - maven-repo-v1-{{ .Branch }}-{{ checksum "ingestion-beam/pom.xml" }}
           - maven-repo-v1-{{ .Branch }}-
           - maven-repo-v1-
```

Comment:
looks like the duplicate should be removed?

Reply:
indeed